src/client/unified-form/validators/cover-tab.ts

```diff
@@ -0,0 +1,52 @@
+import {get, size} from 'lodash';
+import {validateAuthorCreditSection, validateIdentifiers, validateNameSection} from '../../entity-editor/validators/common';
+
+/**
+ * Validates the ISBN Field.
+ *
+ * @param {object} isbn - ISBN state object containing `type` and `value`
+ * @returns {boolean} - true if valid, false if invalid
+ */
+export function validateISBN(isbn) {
+	// since type will already be defined for valid ISBNs
+	return !(
+		Boolean(isbn) &&
+        !get(isbn, 'type', null) &&
+        get(isbn, 'value', '').length > 0
+	);
+}
+
+/**
+ * Validates the Cover Tab state.
+ *
+ * @param {object} data - the form state object
+ * @param {Array} identifierTypes - the list of identifier types
+ * @returns {boolean} - true if form state valid, false if invalid
+ */
+export function validateCoverTab(data:any, identifierTypes:any[]) {
+	return validateNameSection(get(data, 'nameSection')) &&
+    validateIdentifiers(get(data, 'identifierEditor', {}), identifierTypes) &&
+    validateAuthorCreditSection(get(data, 'authorCreditEditor'), true) &&
```

Comment:
Something else we need to fix (not in this PR) is the validation state of the author credit; since we made the author credits a required field, we need to show the validation label like we do for other fields to show it in invalid when empty.